docs/book/src/introduction.md

```diff
@@ -1 +1,90 @@
-# Introduction
+# Kubernetes Cluster API Provider IBM Cloud
+
+<p align="center">
+<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100x">
```

Comment:
wondering some way we can also get the IBMCloud image listed next to k8s, image already exists here - ./docs/images/ibm-cloud.svg

Reply:
@mkumatag , changes made, PTAL.